### ISSUE TITLE
CURA 4.10 does not import profiles or printers from 4.9.1

### ISSUE THREAD

**Issue by Xoxe2** [Ultimaker/Cura#10220]
### Application Version

4.10

### Platform

OS X 10.13.6 High Sierra

### Printer

Personalized Kingroon KP3S

### Reproduction steps

The update to CURA 4.10 does not update the printer or the custom profiles that I have created for each printing assumption, resolution, speeds, retraction, flow, and many more details created in 9.0 and updated to 9.1. Creating a new printer in 4.10 gives me an error when importing these profiles.

### Actual results

![Captura de pantalla 2021-08-02 a las 13 06 15](https://user-images.githubusercontent.com/70057033/127854381-ba0ccaf0-ea7d-449f-975a-926920f82bdc.png)

![Captura de pantalla 2021-08-02 a las 13 25 16](https://user-images.githubusercontent.com/70057033/127854478-946441d7-91ae-4fc1-8cd9-e6add85e33cb.png)

Error al importar el perfil de /Users/xoxeluiz/Documents/perfiles cura/KP3S Low Quality FLOW.curaprofile: El tipo de calidad 'low' no es compatible con la definición actual de máquina activa 'fdmprinter'.

### Expected results

Import custom files 

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Additional information & file uploads

![Captura de pantalla 2021-08-02 a las 13 30 59](https://user-images.githubusercontent.com/70057033/127855221-da6b8720-b11b-47e5-90d9-ec8fceeb4b07.png)

**Comment by fvrmr** [comment]
Hi @Xoxe2 this has to do with that your printer doesn't have the low quality profile. And the profile you want to import is based on that profile.
You can find here more information about importing profiles: #8229 
Did you create these profiles yourself in 4.9?

**Comment by Xoxe2** [comment]
Yes. Create a new Kongroon KP3S printer indicating the parameters and the start and end g-codes.
![Captura de pantalla 2021-08-02 a las 17 33 57](https://user-images.githubusercontent.com/70057033/127886931-9559b2fa-63b8-4b64-b783-20b90e02dc68.png)
![Captura de pantalla 2021-08-02 a las 17 34 16](https://user-images.githubusercontent.com/70057033/127886940-6efd90e5-4075-4472-9d15-afc6ecfd793b.png)

**Comment by Xoxe2** [comment]
I have solved the CURA 4.10 problem by creating a printer profile with the "Creality Ender 2" associated with the "nozzle" and modifying its code to suit the KP3S specifications and changing the name of the printer to my liking. I have installed a custom dock for the KP3S in the "meshes" folder.
All this within the CURA application, in the "resources" folder and each file in the corresponding folder. OS X (Mac)
I have inserted the starting and ending g-codes directly into CURA.
I have been able to recover all my old custom files,
